src/GrumPHP/Task/Git/CommitMessage.php

```diff
@@ -91,6 +93,8 @@ public function run(ContextInterface $context)
                 $regex->addPatternModifier('m');
             }
 
+            array_map(array($regex, 'addPatternModifier'), str_split((string) $config['additional_modifiers']));
```

Comment:
Currently the tests fail because when the additional modifiers are empty, `str_split` is returning an array with a empty string. You will have to add `array_filter` around the str_split method or an is-empty check around the array_map method.